Logos/AristaAwakeSecurity.svg

```diff
@@ -0,0 +1,758 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg
+   xmlns:dc="http://purl.org/dc/elements/1.1/"
+   xmlns:cc="http://creativecommons.org/ns#"
+   xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+   xmlns:svg="http://www.w3.org/2000/svg"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   version="1.1"
+   id="Layer_1"
```

Comment:
id should be GUID and no xlinks should be there. You can access [guidelines](https://microsoft.sharepoint.com/:x:/r/teams/AzureSecurityInsights/_layouts/15/guestaccess.aspx?e=aETsaQ&share=EWs_HuWia89Ns0pc1jXRWQkB7aNXX6Zi1REQcKtH4gQp5Q&isSPOFile=1). Please update it.

Reply:
This has been fixed